modules/apps/object/object-test/src/testFunctional/tests/ObjectForms.testcase

```diff
@@ -613,8 +613,8 @@ definition {
 				fieldValue = "Entry test");
 
 			AssertClick(
-				locator1 = "Button#SUBMIT_FOR_WORKFLOW",
-				value1 = "Submit for Workflow");
+				locator1 = "Button#SAVE",
+				value1 = "Save");
```

Comment:
Hey @larissacribeiro, can we relate this change in the UI to any task mapped by our team?

Are we sure this is the new expected behavior?

Reply:
Hey @igor-franca! yes, it is the new behavior, you can see the related bug [here](https://liferay.atlassian.net/browse/LPD-35127).

This change broke many other tests, which were fixed on [LPD-37725](https://liferay.atlassian.net/browse/LPD-37725), and We just missed fixing this one too.